docs/user/build-customization.rst

```diff
@@ -119,7 +119,7 @@ You can use this approach to cancel builds that you don't want to complete based
    `the Unix implementation does this automatically <https://tldp.org/LDP/abs/html/exitcodes.html>`_
    for exit codes greater than 255.
 
-   .. code-block:: pycon
+   ::
```

Comment:
Didn't this work with `pycon`?

Reply:
It was giving me a `black` error, but I thought `pycon` was a typo :D

Reply:
Yes, I thought so too a long time ago... but it's a real thing :) https://pygments.org/docs/lexers/#pygments.lexers.python.PythonConsoleLexer

Reply:
```suggestion
   .. code-block:: pycon
```